package.json

```diff
@@ -31,7 +31,8 @@
   },
   "homepage": "https://github.com/1-Platform/one-platform#readme",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "cli-microservice": "cd packages && yo --no-insight --no-update-notifier ../cli/generator-op-microservice/generators/microservice"
```

Comment:
Can you create an alias/bash file for this command inside packages/cli.
That way, you just have to execute one file instead of a long command.